### PR TITLE
Add a role for configuring Trackster

### PR DESCRIPTION
**Pull request by afgane**
Doing this as a dedicated PR since I'm not sure why Trackster hasn't been configured already.

### CODE REVIEW

**Comment by mvdbeek** [comment]
I didn't even know you had to configure trackster, though I'm almost never using it.
Looks good, but can you add it as a submodule?

**Comment by afgane** [comment]
Ahh  yes, although this is another thing I wanted to ask you - would you be ok if we transition from including the roles / using submodules into the repo to using `requirements_roles.yml` file that can then be installed via the following command: `ansible-galaxy install -r requirements_roles.yml -p roles`? I've updated this PR with this so you see what I mean and if you're ok with it, I can move the rest of the roles into that file. The file can be used to specify exact revisions and branches as well, so it's like the submodules in many ways but much lighter weight, I feel.

**Comment by mvdbeek** [comment]
I think that's much nicer for users. Lot's of things can (and will) go wrong with submodules. Yes, I like it. Can you add the command to the .travis.yml ?

**Comment by drosofff** [comment]
@afgane I like it too ! Are all the submodules roles deposited to https://galaxy.ansible.com ?

**Comment by afgane** [comment]
That's the beauty - they don't have to be (but they can if that's the preference). They can be pulled directly from their native Github repos  so we can skip the step of needing to publish every role in Galaxy Ansible (and risk those being out of sync). 

BTW, where in `.travis.yml` should I add the install command? I can't find the clone command (https://travis-ci.org/ARTbio/GalaxyKickStart/builds/155321737#L218) anywhere in the repo?

**Comment by afgane** [comment]
OK, that wasn't it. I'll wait until I hear back from you guys on where to add the install command as I don't see the preceding steps in the Travis config so not sure if it's done in the service settings or where else.

File: .travis.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ before_install:
   - sudo su $GALAXY_TRAVIS_USER -c 'tar xfz master.tar.gz'
 
 install:
+  - ansible-galaxy install -r requirements_roles.yml -p roles
```
Comment (mvdbeek) [inline .travis.yml]: @afgane this would need to move before we start building the docker container on line 36.
If you put this at line 31 it should work.

**Comment by mvdbeek** [comment]
> BTW, where in .travis.yml should I add the install command? I can't find the clone command (https://travis-ci.org/ARTbio/GalaxyKickStart/builds/155321737#L218) anywhere in the repo?

travis does the cloning, the current working directory is the base of this repo.

**Comment by mvdbeek** [comment]
Argh, it's missing ansible, `pip install ansible` should make it work.

**Comment by afgane** [comment]
Rebased to get rid of the experimental steps...

**Comment by mvdbeek** [comment]
Very nice, I like it 👍 . I'll leave the honour to @drosofff

**Comment by drosofff** [comment]
Thanks @afgane 👍 . We'll see at doing the same for the other roles.

**Pull request merged**